packages/web-worker/package.json

```diff
@@ -29,6 +32,10 @@
       "types": "./build/ts/webpack-parts/index.d.ts",
       "default": "./webpack.js"
     },
+    "./webpack-loader.js": {
+      "types": "./build/ts/webpack-parts/loader.d.ts",
+      "default": "./build/cjs/webpack-parts/loader.js"
```

Comment:
This being the only entry whose `default`  value comes from the `./build` folder, I'm assuming it's correct, but I just wanted to confirm it wasn't an accident :)